core/common/boot/src/test/java/org/eclipse/edc/boot/system/ExtensionLoaderTest.java

```diff
@@ -98,6 +98,36 @@ void loadMonitor_whenNoMonitorExtension() {
         assertTrue(monitor instanceof ConsoleMonitor);
     }
 
+    @Test
+    void loadMonitor_programArgsSetConsoleMonitorLogLevel() {
+
+        var monitor = ExtensionLoader.loadMonitor(new ArrayList<>(), ConsoleMonitor.LEVEL_PROG_ARG + "=INFO");
+
+        var expectedMonitor = new ConsoleMonitor(ConsoleMonitor.Level.INFO, true);
+
+        assertEquals(monitor, expectedMonitor);
+    }
+
+    @Test
+    void loadMonitor_consoleMonitorDefaultLogLevelWhenNoArgs() {
+
+        var monitor = ExtensionLoader.loadMonitor(new ArrayList<>());
+
+        var expectedMonitor = new ConsoleMonitor(ConsoleMonitor.Level.getDefaultLevel(), true);
+
+        assertEquals(monitor, expectedMonitor);
+    }
+
+    @Test
+    void loadMonitor_consoleMonitorDefaultLogLevelWhenWrongArgs() {
```

Comment:
as I already said, this should result in an exception thrown.

Reply:
[638be6a](https://github.com/eclipse-edc/Connector/pull/4476/commits/638be6acf4ced6b1909be5e3c50988cbedc7f2a0)